__tests__/reactTags.test.js

```diff
@@ -562,10 +562,10 @@ describe('Test ReactTags', () => {
         })
       );
       const $input = $el.find('.ReactTags__tagInputField');
-      $input.simulate('change', { target: { value: 'Ea' } });
+      $input.simulate('change', { target: { value: 'Or' } });
```

Comment:
revert this to `Ea` and change the expect result in line 568

Reply:
Accepted